app-torcx/docker/files/docker-1.12.6-manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "kind": "image-manifest-v0",
+    "value": {
+        "bin": [
+            "/bin/containerd",
+            "/bin/containerd-shim",
+            "/bin/ctr",
+            "/bin/docker",
+            "/bin/docker-containerd",
+            "/bin/docker-containerd-shim",
+            "/bin/docker-proxy",
+            "/bin/docker-runc",
+            "/bin/dockerd",
+            "/bin/runc"
+        ],
+        "service": [
+            "/lib/systemd/system/containerd.service",
+            "/lib/systemd/system/docker.service",
+            "/lib/systemd/system/docker.socket",
```

Comment:
Ignition configuration uses the term "units" for this to cover specifying any of `.service`, `.socket`, `.mount`, etc.  I think that would work here, too.